pkg/vault/vault_test.go

```diff
@@ -143,7 +143,7 @@ func TestGetVaultKey(t *testing.T) {
 			"api_key",
 			ansible_vault.DecryptFile,
 			"",
-			errors.New("yaml: unmarshal errors:\n  line 3: cannot unmarshal !!str `I'm not...` into map[string]string"),
+			errors.New("yaml: unmarshal errors:\n  line 3: cannot unmarshal !!str `I'm not...` into map[string]interface {}"),
```

Comment:
👍 

Can you add test matching your switch case ?

```golang
		{
			"boolean string",
			"secret",
			"./",
			path.Join(ansibleFolder, "sanitized_vault.yml"),
			"integer",
			ansible_vault.DecryptFile,
			"11",
			nil,
		},
```

Reply:
Yes, done. (Y)